build_android.sh

```diff
@@ -0,0 +1 @@
+flutter build appbundle --no-sound-null-safety lib/v2/main.dart
```

Comment:
So i don't always have to type it, I never remember where the null safety param is going. This builds the android app bundle for shipping to Google Play. 

Happy to move it elsewhere... but I need it so I would def. check it in, since otherwise, I can either add it to .gitignore, creating more "mystery" for others who download our repo (like "oh I need to run with no sound null safety")

That being said, this could also go in the README - I don't mind.

Reply:
I put it in readme now - LMK if this is offensive where it is, I can remove it ;)

Reply:
What is the purpose of this PR? 
Prs should be single purpose. I think this should be on a different PR where we can talk about it in isolation.

Reply:
fine. File removed.